api/appendix_h.asciidoc

```diff
@@ -164,6 +164,22 @@ When Device-Side Enqueue is not supported:
 
 |====
 
+When Device-Side Enqueue is supported but a replaceable On-Device Queue is not supported:
+
+[cols="2,3",options="header",]
+|====
+|*API*
+|*Behavior*
+
+| {clGetDeviceInfo}, passing +
+{CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES}
+| May not set {CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT} , indicating that _device_ does not support a replaceable default On-Device Queue.
```

Comment:
I'd suggest a slight wording change here: "may omit CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT" instead of "may not set".  When I read "may not set" I initially think that it is prohibited, not that it is optional.

Reply:
Done